### PR TITLE
Logan difficulty

### PR DESCRIPTION
**Pull request by exasperlnc**
difficulty selection mostly works, easy selection causes the computer to place two letters at once for some reason, lol.

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -51,7 +52,7 @@ def render(message='')
     puts message
   end
 
-  def place(letter, col)
+  def place(letter, col = rand(0..6))
```
Comment (abwmn) [inline lib/board.rb]: this should stay (letter, col), and the easy pick function will perform the rand loop until it gets an empty column

File: lib/game.rb [hunk 2]
```diff
@@ -73,7 +101,30 @@ def prompt
     end
   end
 
-  def pick
+  def easy_pick
+    letter = 'O'
```
Comment (abwmn) [inline lib/game.rb]: col = nil
      loop do
        col = rand(0..6)
        node = @grid[col].find {|node| node.empty? }
        break if node
      end
      col

**Review by abwmn — CHANGES_REQUESTED** [review]
this looks good. the easy pick method should perform the find random empty column check, and pass it to pick pre-validated. as is, there is no check if the random column is empty.

**Review by exasperlnc — COMMENTED** [review]
reverted place change, fixed easy pick

**Pull request merged**